sdk/communication/azure-communication-chat/src/main/java/com/azure/communication/chat/ChatThreadAsyncClient.java

```diff
@@ -762,13 +765,54 @@ public String getChatThreadId() {
     /**
      * Posts a typing event to a thread, on behalf of a user.
      *
+     * @param options Options for sending the typing notification.
```

Comment:
TypingNotificationOptions ?

Reply:
In the comment? Just options is the name of the variable, and consistent with the rest of the options in other existing methods